configs/envoyproxy_io_proxy.yaml

```diff
@@ -0,0 +1,55 @@
+admin:
```

Comment:
Are you planning on nuking the google_com YAML? SGTM if so.

Reply:
i was wondering - which was part of the reason for marking it WIP - to reckon i should check through a bit further - and happy to remove

Reply:
Yep, there is no good reason to be pointing at google.com, it was just arbitrary.